docs/guide/dev/deep_dive/errors/README.md

```diff
@@ -23,6 +23,18 @@ This article contains the list of CKEditor error codes and their explanation. Th
 * Description: Incorrect {@linkapi CKEDITOR.config#autoEmbed_widget } value. No widget definition found.
 * Additional data: None.
 
+## cloudservices-no-url
+
+* Location: `plugins/cloudservices/plugin.js`
+* Description: Upload URL used by {@linkapi CKEDITOR.plugins.cloudServices.cloudServicesLoader} is not set. An URL can be provided by configuration {@linkapi CKEDITOR.config#url} property or passed to [Cloud Services](https://ckeditor.com/cke4/addon/cloudservices) plugin {@linkapi CKEDITOR.plugins.cloudServices.cloudServicesLoader#upload} method.
+* Additional data: None.
+
+## cloudservices-no-token
+
+* Location: `plugins/cloudservices/plugin.js`
+* Description: Authentication token used by {@linkapi CKEDITOR.plugins.cloudServices.cloudServicesLoader} is not set. An token can be provided by configuration {@linkapi CKEDITOR.config#cloudServices_token} property or passed to [Cloud Services](https://ckeditor.com/cke4/addon/cloudservices) plugin {@linkapi CKEDITOR.plugins.cloudServices.cloudServicesLoader#constructor}.
```

Comment:
"An token" -> "The token"

Reply:
Same thing that it should be more generic.

Reply:
There's a small issue that we don't have yet the docs on setting Cloud Services token. So we don't ave anything to point to ATM, we'll add it later.